### PR TITLE
Move topics into overrides and update docs

### LINKED ISSUE #90: Update directory address in kafka topic instructions
The instructions to [create new Kafka topics](https://github.com/operate-first/apps/blob/master/docs/odh/kafka/add_kafka_topics.md) need to be updated after the recent directory restructure.

### PR DESCRIPTION
**Pull request by HumairAK**
Also fixes #90 
Related #87 

Also added some example topics for crc, and quicklab deployments so the topics folder isn't sitting empty.

### CODE REVIEW

**Review by anishasthana — APPROVED** [review]
No complaints here.

File: odh/overlays/moc/kafka/kustomization.yaml [hunk 1]
```diff
@@ -4,17 +4,18 @@ kind: Kustomization
 
 resources:
   - ../../../base/kafka
-  - topics
 
 patchesJson6902:
   - patch: |
```
Comment (4n4nd) [inline odh/overlays/moc/kafka/kustomization.yaml:9]: @HumairAK This patch only adds the topics overlay. Do we need something that also deploys the base? 
i.e. the [kafka cluster](https://github.com/operate-first/apps/blob/7f876c9a36e24c8c0502ecebbf6f2a7438a8d48b/odh/overlays/moc/kafka/overrides/kafka/base/kafka-cluster.yaml)?

Comment (4n4nd) [reply odh/overlays/moc/kafka/kustomization.yaml:9]: I think same would go for other environments as well

Comment (4n4nd) [reply odh/overlays/moc/kafka/kustomization.yaml:9]: nvm it deploys the base as well. Just tested :+1:

**Review by 4n4nd — APPROVED** [review]
LGTM :+1:

**Pull request merged**